invoices/invoiceregistry.go

```diff
@@ -621,15 +645,17 @@ func (i *InvoiceRegistry) NotifyExitHopHtlc(rHash lntypes.Hash,
 		debugLog("expiry too soon")
 
 		return &HodlEvent{
-			Hash: rHash,
+			Hash:           rHash,
+			FinalCltvDelta: expiry - uint32(currentHeight),
```

Comment:
is primary reason for doing this that it doesn't require database changes? and defer storing `accepted height` to do in one refactor?

Reply:
`AcceptedHeight` is complicated because there can be multiple held htlcs with different accept heights. This is meant as a stop gap until we have an invoice registry that tracks each htlc paying to an invoice.